client/state/selectors/is-site-wpforteams-hub.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+import isSiteWPForTeams from 'calypso/state/selectors/is-site-wpforteams';
+
+/**
+ * Returns true if site is a WP for Teams hub, false if not and null if unknown.
+ *
+ * @param  {object}   state  Global state tree
+ * @param  {number}   siteId Site ID
+ * @returns {?boolean}        Whether site is a WP for Teams site hub
+ */
+export default function isSiteWPForTeamsHub( state, siteId ) {
```

Comment:
I suppose we could start using the P2 naming instead of wp for teams for the new code.